doc/manual/team.rst

```diff
@@ -224,6 +225,7 @@ TOO-LATE
 
 The judges may have prepared multiple test files for each problem.
 DOMjudge will report back the first non-correct result as verdict.
+*DOMjudge will always output the highest priority verdict.*
```

Comment:
Maybe add something like “if there are multiple verdicts with the same priority, the first encountered one will be returned”?